src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import { FilterOptions } from '../../types/FilterOptions';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  currentFilter: FilterOptions;
+  onFilterOptionChange: (status: FilterOptions) => void;
+  onCompletedDelete: () => void;
+}
+
+export const Footer: FC<Props> = ({
+  todos,
+  currentFilter,
+  onFilterOptionChange,
+  onCompletedDelete,
+}) => {
+  const todosLeft = todos.reduce(
```

Comment:
With filter it a bit more readable
```suggestion
 const todosLeft = todos.filter(todo => !todo.completed).length;

```